test/Microsoft.AspNetCore.Mvc.FunctionalTests/DataAnnotationTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using RazorWebSite;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.FunctionalTests
+{
+    public class DataAnnotationTests : IClassFixture<MvcTestFixture<StartupDataAnnotations>>
+    {
+        private MvcTestFixture<StartupDataAnnotations> _fixture;
+
+        public DataAnnotationTests(MvcTestFixture<StartupDataAnnotations> fixture)
+        {
+            _fixture = fixture;
```

Comment:
Here's the recommended way to do this

```C#
var factory = fixture.Factories.FirstOrDefault() ?? fixture.WithWebHostBuilder(builder => builder.UseStartup<StartupDataAnnotations>());
Client = factory.CreateDefaultClient();
```
You can remove the `ConfigureServices` call that way